internal/services/springcloud/spring_cloud_service_resource.go

```diff
@@ -581,6 +603,22 @@ func resourceSpringCloudServiceRead(d *pluginsdk.ResourceData, meta interface{})
 	if utils.ResponseWasNotFound(serviceRegistry.Response) {
 		serviceRegistryEnabled = false
 	}
+
+	defaultBuildServiceEnabled := true
+	if resp.Sku != nil && resp.Sku.Name != nil && *resp.Sku.Name == "E0" {
```

Comment:
why are we checking the sku here?

Reply:
Build service is only supported in enterprise sku.